### PR TITLE
Use consistent aside styling for doc/article pages

### PR DESCRIPTION
**Pull request by mportiz08**
Bug/issue #, if applicable: 89588748

## Summary

This updates the aside styling for tutorial-style article pages to match the same styling used in normal documentation pages. This does not impact the more minimal styling used for tutorial pages themselves.

Before:
Screenshot 2023-03-27 at 4 41 48 PM

After:
Screenshot 2023-03-27 at 4 41 52 PM

## Testing

Steps:
1. Run `npm run build` to build this branch.
2. Configure `DOCC_HTML_DIR=/path/to/dist` when previewing example content with DocC.
3. Verify that documentation pages and tutorial article pages both use the same styling for asides.
4. Verify that tutorial pages still use the more minimal existing styling for asides.

[Example.docc.zip](https://github.com/apple/swift-docc-render/files/11083807/Example.docc.zip)

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] ~~Added tests~~ (CSS-only change)
- [X] Ran `npm test`, and it succeeded
- [X] Updated documentation if necessary

### CODE REVIEW

**Comment by SamLanier** [comment]
What does this look like when an aside is in a step card?

**Comment by dobromir-hristov** [comment]
> What does this look like when an aside is in a step card?

Tutorials should not be affected, only "Tutorial Articles", which are a completely different page type.

**Comment by SamLanier** [comment]
> > What does this look like when an aside is in a step card?
> 
> Tutorials should not be affected, only "Tutorial Articles", which are a completely different page type.

But we want this style for tutorial intro sections, just not step cards.

**Comment by dobromir-hristov** [comment]
> But we want this style for tutorial intro sections, just not step cards.

@SamLanier if it has to be in the Overview pages as well, then @mportiz08 has to add an extra class.

**Comment by SamLanier** [comment]
> > But we want this style for tutorial intro sections, just not step cards.
> 
> @SamLanier if it has to be in the Overview pages as well, then @mportiz08 has to add an extra class.

Yeah, we've see authors use them in the **Hero** intro sections as well as **Section** intros. Both of these would be valid places to add a regular-style aside. Then in the step cards, the aside should be more of a compact-aside style.

**Comment by mportiz08** [comment]
@swift-ci test

**Comment by mportiz08** [comment]
I refactored things to invert the aside styling so that _only_ asides within a `@Steps` directive get the more minimal/compact styling and all other scenarios look like the more common aside styling seen on documentation pages today.

Let me know if that sounds right!

File: src/components/ContentNode/Aside.vue [hunk 1]
```diff
@@ -77,18 +60,41 @@ $aside-kinds: deprecated, experiment, important, note, tip, warning;
...
+        color: var(--color-aside-#{$kind});
+      }
     }
   }
+}
 
-  .label {
-    @include font-styles(documentation-aside-name);
+// Apply a more compact styling for asides within a `@Steps` directive
+.steps {
```
Comment (dobromir-hristov) [inline src/components/ContentNode/Aside.vue:87]: Should we do this from within the `Step.vue` or `SectionStep.vue` components? I am fine keeping it here, just thinking out loud.

**Review by dobromir-hristov — COMMENTED** [review]
Here are a few screens of example pages for Sam to see. We may want to explicitly state the `text-align` property, see how the Overview page centers the text.


image
image
image
image

**Comment by SamLanier** [comment]
Thanks @dobromir-hristov, this looks great. What happens if you stick one here, next to "Some text.":
Screenshot 2023-03-29 at 9 13 35 AM

**Comment by mportiz08** [comment]
> Thanks @dobromir-hristov, this looks great. What happens if you stick one here, next to "Some text.":
> Screenshot 2023-03-29 at 9 13 35 AM

Screenshot 2023-03-29 at 4 03 02 PM

File: src/components/ContentNode/Aside.vue (hunk 1 above)
Comment (mportiz08) [reply src/components/ContentNode/Aside.vue]: No preference, but I moved it 5c15fb3ae1d59826363b79167716dbf5f23020d4

**Comment by mportiz08** [comment]
> Here are a few screens of example pages for Sam to see. We may want to explicitly state the `text-align` property, see how the Overview page centers the text.

89da97bb4e98eb94847de8f1709ad77d089c750a

**Comment by mportiz08** [comment]
@swift-ci test

File: src/components/ContentNode/Aside.vue [hunk 2]
```diff
@@ -76,6 +59,15 @@ $aside-kinds: deprecated, experiment, important, note, tip, warning;
     $aside-width-border
     $aside-width-left-border);
   padding: rem(16px);
+  text-align: start;
```
Comment (dobromir-hristov) [inline src/components/ContentNode/Aside.vue:62]: image

**Review by dobromir-hristov — APPROVED** [review]

**Review by SamLanier — APPROVED** [review]
This is exactly what I expect asides to look like in each scenario. Design approved.

**Comment by mportiz08** [comment]
@swift-ci test

**Comment by hqhhuang** [comment]
@swift-ci test

**Pull request merged**